db/seeds.rb

```diff
@@ -1,8 +1,455 @@
 # frozen_string_literal: true
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+
+STUDENTS_LENGTH = rand(20..40).to_i
+
+students = Student.create!(Array.new(STUDENTS_LENGTH) { { name: Faker::Name.name, born_on: 18.years.ago } })
+teachers = Teacher.create!(Array.new(7) { { name: Faker::Name.name } })
+subjects = Subject.create!([
+                             {
+                               name: 'Matemática'
+                             },
+                             {
+                               name: 'Português'
+                             },
+                             {
+                               name: 'História'
+                             },
+                             {
+                               name: 'Geografia'
+                             },
+                             {
+                               name: 'Física'
+                             },
+                             {
+                               name: 'Química'
+                             },
+                             {
+                               name: 'Inglês'
+                             }
+                           ])
+
+courses = Course.create!([{
```

Comment:
Again, the same here, use the tools to your advantage. It's not wrong, but it's way too long. You can observe a pattern: Each year has the same amount of courses, with the same date for starts_on and ends_on, only the year changes.